src/commands/suggestions/suggestionList.ts

```diff
@@ -0,0 +1,57 @@
+import { Message, MessageEmbed } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+
+import { AdminCommand } from '../../utils/commands';
+import { availableLists, getAvailableListsString, getSuggestions, Suggestion } from '../../components/suggestions';
+import { EMBED_COLOUR } from '../../utils/embeds';
+import { parseListArg, validateListArg } from './utils';
+
+const RESULTS_PER_PAGE = 6;
+
+class SuggestionsListCommand extends AdminCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'suggestions-list',
+      aliases: ['suggestions', 'suggestion-list', 'suggestions-list'],
+      group: 'suggestions',
+      memberName: 'list',
+      args: [
+        {
+          key: 'list',
+          prompt: `enter one of ${getAvailableListsString()}.`,
+          type: 'string',
+          default: '',
+          validate: validateListArg,
+          parse: parseListArg
+        }
+      ],
+      description: 'Shows you a list of suggestions.',
+      examples: [`${client.commandPrefix}suggestions`, `${client.commandPrefix}suggestions-list actionable`]
+    });
+  }
+
+  private async getSuggestionDisplayInfo(suggestion: Suggestion) {
+    return '**' + suggestion['id'] + '** | ' + suggestion['state'] + ' | ' + suggestion['suggestion'] + '\n\n';
+  }
+
+  async onRun(message: CommandoMessage, args: { list: string }): Promise<Message> {
+    const { list } = args;
+
+    // query suggestions
+    const suggestions = await getSuggestions(args.list.toLowerCase());
```

Comment:
don't think we need to do `toLowerCase` here because our parser does that for our arguments, still check to make sure though

Reply:
also since you already have `const { list } = args` you don't need to do `args.list` anymore, and I think we should call this `state` instead of `list`